src/services/content/hooks/redirect.js

```diff
@@ -0,0 +1,24 @@
+const { authenticate } = require('@feathersjs/authentication');
+const { disallow } = require('feathers-hooks-common');
+
+exports.before = {
+	all: [
+		authenticate('jwt'),
+	],
+	find: [disallow()],
+	get: [],
```

Comment:
Please add a comment if it is ok, that no scope hooks exist.

Reply:
@kremer-io can you say something about this?